zsl/application/service_application.py

```diff
@@ -135,7 +140,18 @@ def get_version(self):
         else:
             return AtteqServiceFlask.VERSION + ":" + v
 
-    run_web = Flask.run
+    def run_web(self, host='127.0.0.1', port=5000, debug=False, **options):
+
+        options.setdefault('use_debugger', self.config.get('USE_DEBUGGER', False))
+        options.setdefault('use_reloader', self.config.get('USE_RELOADER', False))
+
+        return self.run(
+            host=self.config.get('FLASK_HOST', host),
+            port=self.config.get('FLASK_PORT', port),
+            debug=self.config.get('DEBUG', debug),
+            **options
+        )
+
```

Comment:
I moved the default configuration from `run` module here to enable simple `app.run_web()` in client code.